client/python/ovmsclient/lib/scripts/build_tfs_api.sh

```diff
@@ -18,6 +18,15 @@
 
 set -e
 
+finally() {
```

Comment:
Shouldn't it be:
`function finally() {`
?
Also I would change it's name to something like "remove_tmp_dirs" as "finally" looks a bit like some keyword or special value.

Reply:
God point, make function name more descriptive

Reply:
cleanup ?
final_cleanup ?

Reply:
function keyword is not required. Changed the function name to be more descriptive.